src/Native/Windows/gen-buildsys-win.bat

```diff
@@ -10,9 +10,8 @@ if %1=="/?" GOTO :USAGE
 
 setlocal
 set __sourceDir=%~dp0
-:: Default to vs2013 unless vs2015 is specified
-set __VSString=12 2013
-if /i "%2" == "vs2015" (set __VSString=14 2015)
+:: VS 2015 is the minimum supported OS
+set __VSString=14 2015
```

Comment:
I see a few straggling references in the repro to 2013, e.g.
https://github.com/dotnet/coreclr/blob/master/Documentation/building/debugging-instructions.md
We should clean those up if we're now saying VS2015 is required.